boards/t420/t420.config

```diff
@@ -1,11 +1,23 @@
-# Configuration for a T420 running Qubes and other OS, T420 is identical to X230 on the Linux Side of things.
+# Configuration for a T420 running Qubes and other Linux Based OSes (through kexec)
+#
+# Includes
+# - Deactivated+neutered ME and expended consequent IFD BIOS regions
+# - Forged 00:DE:AD:C0:FF:EE MAC address  (if not extracting gbe.bin from backup with blobs/xx30/extract.sh)
```

Comment:
xx20

Reply:
Corrected.  Also corrected in x220.config